go.mod

```diff
@@ -32,4 +33,4 @@ require (
 	k8s.io/client-go v0.17.9
 )
 
-replace github.com/osrg/gobgp v2.0.0+incompatible => github.com/osrg/gobgp v0.0.0-20191101114856-a42a1a5f6bf0
+//replace github.com/osrg/gobgp v2.0.0+incompatible => github.com/osrg/gobgp v0.0.0-20191101114856-a42a1a5f6bf0
```

Comment:
Thanks for the contribution @zvfvrv ! Could you just remove this comment, and I'll merge this PR :)

Reply:
Done!